frontend/src/components/BackButton.tsx

```diff
@@ -0,0 +1,11 @@
+import backIcon from '../../images/backIcon.png'
+const BackButton = () => {
+  return (
+    <button style={{ width: '45px', height: '45px', display: 'inline-flex', alignItems: 'center', justifyContent: 'center' }}
+    className={"rounded-full bg-secondary-bg-500"}>
+    <img src={backIcon} alt="Back" style={{ width: '60%', height: '35%', filter: 'invert(100%)'}} className="fill-white"/> 
```

Comment:
indentation pls (enable prettier as well so that this can be formatted well - if can't then run `npx prettier . -w`